src/ui/utils/number.ts

```diff
@@ -0,0 +1,5 @@
+export const ordinal = (num: number) => {
+  const mod = num % 10
+  const modMap: Record<number, string> = {1: 'st',2: 'nd',3: 'rd'}
+  return num >10 && num < 14 ? 'th' : (modMap[mod] ?? 'th')
+}
```

Comment:
We dont need separate files like this, just organise them in a single utils file

Reply:
ordinal is not Date specific utils. Also, in the future, this file can be the container for another helper that is related to number

Reply:
Id much rather have all utils in the same file, if it helps just use a class with static methods to segregate within that file. If a specific type has several helpers in the future, we can consider moving that to its own file

Reply:
Its merged in src\utils.ts